nautobot/docs/configuration/optional-settings.md

```diff
@@ -367,6 +366,15 @@ The value of this variable can also be customized by setting the environment var
 
 ---
 
+## GIT_SSL_NO_VERIFY
+
+Default: Unset
+
+If you are using a self-signed git repository, you will need to set the environment variable `GIT_SSL_NO_VERIFY="1"`
+in order for the repository to sync.
```

Comment:
This *has* to be an environment variable, not a `nautobot_config.py` variable, correct?

```suggestion
in order for the repository to sync.

!!! warning
    This *must* be specified as an environment variable. Setting it in `nautobot_config.py` will not have the desired effect.
```

Reply:
Correct.